package.json

```diff
@@ -127,7 +127,7 @@
     },
     "dependencies": {
         "@0x/assert": "^3.0.4",
-        "@0x/asset-swapper": "0xProject/gitpkg-registry#0x-asset-swapper-v4.6.0-0ba79b060",
+        "@0x/asset-swapper": "0xProject/gitpkg-registry#0x-asset-swapper-v4.6.0-26ba3fac4",
```

Comment:
**TODO**: update to `development` branch build when 0xProject/0x-monorepo#2702 is merged